File: Javascript/make_a_payment.js
fetch("https://sandbox.api.mxmerchant.com/checkout/v3/payment?echo=true&includeCustomerMatches=false", {
  "method": "POST",
  "headers": {
    "Content-Type": "application/json",
    "Authorization": "Basic dGVzdGFjY291bnQ6ZmFrZXBhc3N3b3Jk"
  },
  "body": "{\"paymentType\":\"Sale\",\"cardAccount\":{\"number\":\"4111111111111111\",\"expiryMonth\":\"06\",\"expiryYear\":\"23\",\"avsZip\":\"30004\",\"cvv\":\"321\"},\"entryClass\":\"WEB\",\"authOnly\":false,\"isAuth\":false,\"isSettleFunds\":true,\"source\":\"API\",\"taxExempt\":false,\"merchantId\":\"000000000\",\"amount\":0.01,\"tenderType\":\"card\",\"cardPresent\":true}"
})
.then(response => {
  console.log(response);
})
.catch(err => {
  console.error(err);
});

File: C#/make_A_Payment.cs
using System;
using RestSharp;

namespace GitHub_Test
{
    class Program
    {
        static void Main(string[] args)
        {
            var client = new RestClient("https://sandbox.api.mxmerchant.com/checkout/v3/payment?echo=true");
            client.Timeout = -1;
            var request = new RestRequest(Method.POST);
            request.AddHeader("Content-Type", "application/json");
            request.AddHeader("Authorization", "Basic e3tVc2VybmFtZX19Ont7UGFzc3dvcmR9fQ==");
            request.AddParameter("application/json", "{\n\t\"merchantId\": \"516158976\",\n\t\"amount\": 45.00,\n\t\"paymentType\": \"sale\",\n\t\"tenderType\": \"cash\"\n\t}\n}\n", ParameterType.RequestBody);
            IRestResponse response = client.Execute(request);
            Console.WriteLine(response.Content);
        }
    }
}
